### ISSUE TITLE
Please handle errors in room.sendAnnouncement

### ISSUE THREAD

**Issue by uzayyli** [haxball/haxball-issues#1979]
@basro Please add error handling to `room.sendAnnouncement` (and probably `sendChat` too?).

If a null or any non-string value is supplied, the function throws an exception which could be easily handled in the core API.

Please handle the following examples so that it doesn't fail the whole room:

`room.sendAnnouncement(3)`
`room.sendAnnouncement(null)`

Thanks

**Comment by dixtel** [comment]
Wrap this this function with own safety checks, it will be simpler and faster for you.

By the way the function signature is 
`sendAnnouncement(msg:String, targetId?:Int, color?:Int, style?:String, sound?:Int)
`

So you cannot pass a number or null as first parameter.

**Comment by uzayyli** [comment]
yeah i could implement my own sanity checks but something this simple throwing an error and failing the whole room seems wrong :) thanks anyway
